handlers/tealium.js

```diff
@@ -66,3 +66,56 @@ export const handler = async (lambdaEvent) => {
     return Promise.resolve('Nothing processed')
   }
 }
+
+export const bqHandler = async (rows) => {
+  if (typeof rows !== "undefined") {
+    const validEvents = [];
+    rows.forEach((row) => {
+      try {
+        validEvents.push(row);
+      } catch (error) {
+        if (process.env.LOG_LEVEL === "debug") {
+          rollbar.error("Event.fromRecord(row) error", error, {
+            row: row,
+          });
+        }
+
+        if (process.env.LOG_LEVEL === "debug") {
+          console.log("BigQuery Event Error: ", error.toString());
+        }
+      }
+    });
+  
+    if (validEvents.length > 0) {
+      try {
+        // send uniquely valid events to Tealium event API
+        const tealiumPOST = bent("https://collect.tealiumiq.com", "POST");
+        const requests = uniqBy(validEvents, "user_id").map((event) =>
+          retry(
+            async (bail) => {
+              const tealium = new BQTealiumEvent(event);
+              const extraParams = {
+                "cp.trace_id": tealium.dataLayer()["tealium_trace_id"],
+              };
+              return tealiumPOST(
+                "/udp/main/2/i.gif",
+                { data: tealium.dataLayer(extraParams) },
+                // tealium.headers({ Cookie: tealium.cookies() })
+              );
+            },
+            { retries: 3 }
+          )
+        );
+        await Promise.all(requests);
+        return `Processed ${requests.length} events.`;
+      } catch (error) {
+        await rollbar.error("Error sending event to Tealium", error);
```

Comment:
Update the error message to better describe the error was with sending a big query row, or user placement.